components/style/values/specified/source_size_list.rs

```diff
@@ -18,9 +18,12 @@ use values::specified::{Length, NoCalcLength, ViewportPercentageLength};
 /// A value for a `<source-size>`:
 ///
 /// https://html.spec.whatwg.org/multipage/#source-size
+#[derive(Debug, PartialEq)]
 pub struct SourceSize {
-    condition: MediaCondition,
-    value: Length,
+    /// https://html.spec.whatwg.org/multipage/#source-size
```

Comment:
There's no need to make these public, nor below. And why is `PartialEq` needed?

Reply:
The normalising_source_densities requires to extract "value" from the SourceSize.
`let length = &source_set.sourceSize.value.clone().unwrap();` 
Hence value has to be public. Also since we are using sourceSize from SourceSizeList above, we needed its members to be public.

Reply:
`PartialEq` was added for unit tests as I remember, I can remove them from here if needed.